packages/transport/test/modules/messages/MessageController.test.ts

```diff
@@ -226,9 +226,7 @@ describe("MessageController", function () {
             await TestUtil.terminateRelationship(sender, recipient);
         });
 
-        test("should not send a message on a terminated relationship", async function () {
-            await expect(TestUtil.sendMessage(sender, recipient)).rejects.toThrow("error.transport.messages.missingOrInactiveRelationship");
-        });
+        test.todo("should be able to send a Notification on a terminated relationship");
```

Comment:
you speak about Notifications in the transport library - the runtime does not know Notifications. should just be named message as the transport libs let's you send EVERY message regardless of the content.